c-host/installer/installer.nsi

```diff
@@ -0,0 +1,187 @@
+/*
+Looking Glass - KVM FrameRelay (KVMFR) Client
+Copyright (C) 2017-2020 Geoffrey McRae <geoff@hostfission.com>
+https://looking-glass.hostfission.com
+
+This program is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License as published by the Free Software
+Foundation; either version 2 of the License, or (at your option) any later
+version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+Place, Suite 330, Boston, MA 02111-1307 USA
+*/
+
+
+;Include  
+!include "MUI2.nsh"
+!include "FileFunc.nsh"
+!include "LogicLib.nsh"
+!include "Sections.nsh"
+
+;Settings
+Name "Looking Glass Host"
+OutFile "..\build\Looking-Glass-Host-Setup.exe" 
+Unicode true
+RequestExecutionLevel admin
+ShowInstDetails "show"
+ShowUninstDetails "show"
+InstallDir "$PROGRAMFILES64\Looking-Glass"
```

Comment:
I prefer to minimize spaces in paths on windows. So would `LookingGlass` be acceptable?

Reply:
$PROGRAMFILES64 will already have a space in > 99% of cases

Reply:
True, however, you could call it via `%programfiles%`, which does not have space. 

Also, that is the default install directory. It can be overridden with a command line argument or during the install, in which case the path might not have a space.

Reply:
Please use the space in the path.

Reply:
Done.